pymdownx/progressbar.py

```diff
@@ -107,11 +106,12 @@
 DEALINGS IN THE SOFTWARE.
 """
 from __future__ import unicode_literals
+from . import util
 from markdown import Extension
-from markdown.inlinepatterns import Pattern, dequote
 from markdown import util as md_util
 from markdown.extensions.attr_list import AttrListTreeprocessor
-from . import util
+from markdown.inlinepatterns import Pattern
```

Comment:
I know some linters may complain about importing multiple objects on the same line, but I personally prefer these to be imported on the same line.